pkg/ebpf/c/tracee.bpf.c

```diff
@@ -77,6 +77,11 @@ int sys_enter_init(struct bpf_raw_tracepoint_args *ctx)
         task_info = init_task_info(tid, 0);
         if (unlikely(task_info == NULL))
             return 0;
+
+        int zero = 0;
+        config_entry_t *config = bpf_map_lookup_elem(&config_map, &zero);
+        if (config != NULL)
```

Comment:
Maybe a `likely` around this conditional?

Reply:
Or `unlikely` and return if NULL.